configs/components/fesom/fesom.yaml

```diff
@@ -330,6 +330,10 @@ runtime_environment_changes:
                                         vars: ${12_step_outputs}
                                         unit: s
                                         rate: 12
+                                -
+                                        vars: ${1_step_outputs}
+                                        unit: s
+                                        rate: 1
```

Comment:
This variable is not defined anywhere and will be reverted back.

Reply:
I added this, because I needed output after each time step to figure out where my model went crazy.